templates/shortcode-receipt.php

```diff
@@ -150,12 +150,12 @@
 						<div class="edd_purchase_receipt_product_name">
 							<?php echo esc_html( $item['name'] ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
-							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
+							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo esc_attr( edd_get_price_option_name( $item['id'], $price_id, $payment->ID ) ); ?></span>
 							<?php endif; ?>
 						</div>
 
 						<?php if ( $edd_receipt_args['notes'] ) : ?>
-							<div class="edd_purchase_receipt_product_notes"><?php echo wpautop( edd_get_product_notes( $item['id'] ) ); ?></div>
+							<div class="edd_purchase_receipt_product_notes"><?php echo wp_kses_post(wpautop( edd_get_product_notes( $item['id'] ) ) ); ?></div>
```

Comment:
@christ0ph3r you can add the $allowedposttags at the top with the other global we already have.

```
global $edd_receipt_args, $allowedposttags;
```

Then just use `$allowedposttags` as the second param in `strip_tags()`.

`<?php echo strip_tags( edd_get_product_notes( $item['id'] ), $allowedposttags; ); ?>`